backends/ref/ceed-ref-restriction.c

```diff
@@ -392,6 +416,9 @@ static inline int CeedElemRestrictionApply_Ref_Core(CeedElemRestriction r, const
                                                              vv);
         }
         break;
+      case CEED_RESTRICTION_POINTS:
```

Comment:
My plan here is to set this up so the ref impl makes a cell-ordered e-vec of all points, but in practice I think we'll restrict one cell at a time.